<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswdWindow</class>
 <widget class="QDialog" name="PasswdWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>266</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>266</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>266</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="leFirst">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>56</y>
     <width>368</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>368</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>368</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSecond">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>144</y>
     <width>368</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>368</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>368</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2019</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>208</y>
     <width>144</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>144</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>144</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>144</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>144</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>144</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFirst">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>16</y>
     <width>368</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>368</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>368</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Server IP address</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSecond">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>104</y>
     <width>368</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>368</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>368</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Port number</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
